engine/src/srvcgi.cpp

```diff
@@ -1816,6 +1816,9 @@ static bool cgi_send_headers(void)
 			case kMCSOutputTextEncodingUTF8:
 				sprintf(t_content_header, "Content-Type: text/html; charset=utf-8\n");
 				break;
+			default:
```

Comment:
Actually I wonder if we can just remove `kMCSOutputTextEncodingUndefined` entirely... it doesn't appear there's a code pat that can give `MCserveroutputtextencoding` that value...